secret/Tiltfile

```diff
@@ -89,6 +89,38 @@ def secret_from_dict(name, namespace="", inputs={}):
     args.extend(["-o=yaml", "--dry-run=client"])
     return local(args, quiet=True, echo_off=True)
 
+def registry_secret(name, namespace="", flags_dict={}):
```

Comment:
can you call this `secret_yaml_registry`, to match the other functions? Also please add it to the readme